docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+
+services:
+  database:
+    build: ./database
+    volumes:
+      - database-data:/var/lib/postgresql/data
+    ports:
+      - "25432:5432"
+
+  backend:
+    build: ./backend
+    ports:
+      - "5000:5000"
+    volumes:
+      - ./backend/code:/var/opt/gis
+
+  frontend:
+    build: ./frontend
+    ports:
+      - "4200:4200"
+    volumes:
+      - ./frontend/code/src:/var/opt/gis/src
+
```

Comment:
Maybe we also could include pgAdmin4 in the docker compose, lateron? I find it quite useful, as it has also the capability to view geographic data (as opposed to DataGrip ... 🙄 ). 

In that case, there would also go under services: 
```
  pgadmin:
    image: dpage/pgadmin4:latest
    ports:
      - "35050:5050"
    restart: always
    env_file:
      - cred_env/pgadmin.env
```

plus maybe an `.env` file with 
```
PGADMIN_DEFAULT_EMAIL="xyz@uni-konstanz.de"
PGADMIN_DEFAULT_PASSWORD="secret"
PGADMIN_LISTEN_PORT=5050
```